.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
     hooks:
     -   id: isort
 -   repo: https://github.com/psf/black
-    rev: 20.8b1
+    rev: 22.3.0
```

Comment:
Nit: this change could be in its own commit.

Reply:
This should belong to the other PR, I reverted this change